editor/src/com/talosvfx/talos/editor/project/FileTracker.java

```diff
@@ -87,9 +87,10 @@ public void trackFile(FileHandle fileHandle, Tracker tracker) {
         entries.put(fileHandle, new FileEntry(fileHandle, tracker));
     }
 
-    public void update() {
-        Array<FileHandle> filesToRemove = new Array<>();
+    Array<FileHandle> filesToRemove = new Array<>();
 
+    public void update() {
+        filesToRemove.clear();
         final FileTab currentTab = TalosMain.Instance().ProjectController().currentTab;
```

Comment:
when debugging with profiler, saw this function was making lot's of array objects